src/main/webapp/stackTrace.js

```diff
@@ -0,0 +1,45 @@
+/**Copyright 2019 Google LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+// add button for stack trace
+addStackTraceBtn = (liElement, logError) => {
```

Comment:
stackTraceButton

Feel free to use complete words.
It's generally recommended.